dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OidcUtil.java

```diff
@@ -246,18 +237,12 @@ public static void createNonceCookie(OidcClientRequest oidcClientRequest, String
         oidcClientRequest.getResponse().addCookie(cookie);
     }
 
-    /**
-     * @param nonceInIDToken
-     * @param clientConfig
-     * @param responseState
-     * @return
-     */
     @Trivial
     public static boolean verifyNonce(OidcClientRequest oidcClientRequest, String nonceInIDToken, ConvergedClientConfig clientConfig, String responseState) {
         String cookieName = HashUtils.getCookieName(ClientConstants.WAS_OIDC_NONCE, clientConfig, responseState);
         String cookieValue = createNonceCookieValue(nonceInIDToken, responseState, clientConfig);
         String oldCookieValue = CookieHelper.getCookieValue(oidcClientRequest.getRequest().getCookies(), cookieName);
-        OidcClientUtil.invalidateReferrerURLCookie(oidcClientRequest.getRequest(), oidcClientRequest.getResponse(), ClientConstants.WAS_OIDC_NONCE);
+        OidcClientUtil.invalidateReferrerURLCookie(oidcClientRequest.getRequest(), oidcClientRequest.getResponse(), cookieName);
```

Comment:
This change, in particular, should solve the WASOidcNonce* cookies from not being deleted.